src/containers/Profile.vue

```diff
@@ -0,0 +1,301 @@
+<template>
+  <div class="profileContainer">
+    <div class="profileContainer-header">
+      <div class="profileContainer-title">
+        <span> Edit Profile</span>
+      </div>
+      <div class="profileContainer-subtitle">
+        <span> get your information up-to-date! </span>
+      </div>
+    </div>
+    <div class="profileContainer-editPage">
+      <onboarding-basic
+        :userName="name"
+        :onboardingData="onboarding"
+        :isEditingProfile="true"
+        @updateBasic="updateBasic"
+      />
+    </div>
+    <div>
+      <button
+        class="profileContainer-button"
+        @click="submitOnboarding"
+        data-cyId="onboarding-finishButton"
+      >
+        Finish
+      </button>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { PropType, defineComponent } from 'vue';
+import OnboardingBasic from '@/components/Modals/Onboarding/OnboardingBasic.vue';
+import { setAppOnboardingData } from '@/global-firestore-data';
+import { getMajorFullName, getMinorFullName, getGradFullName } from '@/utilities';
+import timeline1Text from '@/assets/images/timeline1text.svg';
+import timeline2Text from '@/assets/images/timeline2text.svg';
+import timeline3Text from '@/assets/images/timeline3text.svg';
+
+const timelineTexts = [timeline1Text, timeline2Text, timeline3Text];
+
+const placeholderText = 'Select one';
+const FINAL_PAGE = 3;
+
+export default defineComponent({
+  components: { OnboardingBasic },
+  props: {
+    userName: { type: Object as PropType<FirestoreUserName>, required: true },
+    onboardingData: {
+      type: Object as PropType<AppOnboardingData>,
+      required: true,
+    },
+  },
+  emits: ['onboard', 'cancelOnboarding'],
+  data() {
+    return {
+      currentPage: 1,
+      name: { ...this.userName },
+      onboarding: { ...this.onboardingData },
+    };
+  },
+  computed: {
+    // Display error if a required field is empty
+    isError(): boolean {
```

Comment:
I'm noticing the error text does not show up if an error occurs. Similarly - users can save their data even if there is an error - resulting in the possibility a user has no college/grad program selected and thus an empty reqs bar. Could you either enforce the error code like onboarding or add it to your todo list for a future PR?

![image](https://user-images.githubusercontent.com/25535093/163052989-a8f65dc9-815f-4c56-989b-356880451012.png)